### PR TITLE
validate manifest colors with DevTools's color parsing code

### PR DESCRIPTION
**Pull request by brendankenny**
also fix a bug where `Array.include` was being called (not yet in Node stable)

### CODE REVIEW

File: helpers/manifest-parser.js [hunk 1]
```diff
@@ -15,6 +15,10 @@
  */
 'use strict';
 
+// Required to initialize WebInspector.Color.
+global.WebInspector = global.WebInspector || {};
```
Comment (paulirish) [inline helpers/manifest-parser.js]: should we do the same in network-recorder now?

Comment (brendankenny) [inline helpers/manifest-parser.js]: Yes, good call

**Comment by brendankenny** [comment]
PTAL

File: helpers/network-recorder.js [hunk 2]
```diff
@@ -22,7 +22,7 @@ global.self = global;
 global.Protocol = {
   Agents() {}
 };
-global.WebInspector = {
+global.WebInspector = Object.assign(global.WebInspector || {}, {
```
Comment (paulirish) [inline helpers/network-recorder.js]: I'd much rather keep this simple and do 

```js
global.WebInspector = global.WebInspector || {};
global.WebInspector._moduleSettings = { …
global.WebInspector.moduleSetting = { …
```

Comment (brendankenny) [inline helpers/network-recorder.js]: yes

**Comment by paulirish** [comment]
lgtm

**Pull request merged**